compiler/compiler.go

```diff
@@ -243,7 +243,8 @@ func ReadArchive(filename, path string, r io.Reader, packages map[string]*types.
 	}
 
 	var err error
-	_, packages[path], err = gcimporter.BImportData(token.NewFileSet(), packages, a.ExportData, path)
+	b := bytes.NewReader(a.ExportData)
+	packages[path], err = gcexportdata.Read(b, token.NewFileSet(), packages, path)
```

Comment:
`b` is not a descriptive variable name. That's often okay in short functions, but it doesn't work well in this context.  It's also typically associated with `[]byte` type, not `io.Reader`. I see `r` is already taken and I don't think it's a good idea to reuse it.

I think it's better to either use a longer descriptive name here:

```Go
exportData := bytes.NewReader(a.ExportData)
packages[path], err = gcexportdata.Read(exportData, token.NewFileSet(), packages, path)
```

Which would mirror the `gcexportdata.Write ` code below.

But I think this line is not that long, that it's even better to just inline it:

```Go
packages[path], err = gcexportdata.Read(bytes.NewReader(a.ExportData), token.NewFileSet(), packages, path)
```

Either of those is okay.

Reply:
Done.